Code/GraphMol/Kekulize.cpp

```diff
@@ -503,19 +504,25 @@ void kekulizeFused(RWMol &mol, const VECT_INT_VECT &arings,
 namespace MolOps {
 namespace details {
 void KekulizeFragment(RWMol &mol, const boost::dynamic_bitset<> &atomsToUse,
-                      const boost::dynamic_bitset<> &bondsToUse,
+                      const boost::dynamic_bitset<> &inBondsToUse,
```

Comment:
Should we just pass this by value, `boost::dynamic_bitset<> bondsToUse`, instead of making a copy after the preconditions?